github/MainClass.pyi

```diff
@@ -32,14 +30,15 @@ class Github:
         login_or_token: Optional[str] = ...,
         password: Optional[str] = ...,
         jwt: Optional[str] = ...,
+        app_id: Optional[Union[int, str]] = ...,
+        app_private_key: Optional[str] = ...,
         base_url: str = ...,
         timeout: int = ...,
-        client_id: Optional[str] = ...,
-        client_secret: Optional[str] = ...,
         user_agent: str = ...,
         per_page: int = ...,
         verify: bool = ...,
         retry: Any = ...,
+        pool_size: Optional[int]=None,
```

Comment:
Should be ` = ...,` to not repeat the default value (see all other arguments in that signature).

Reply:
Same in `Requester.pyi`.